src/jmcomic/jm_option.py

```
@@ -328,6 +330,52 @@ def compatible_with_old_versions(cls, dic):
+                if 'level' not in kwargs:
+                    # 旧版本默认值是 'photo'
+                    level = 'photo'
+                else:
+                    level = kwargs.pop('level')
+
+                if group == 'after_album' and level != 'album':
+                    # after_album + level=photo → 等价迁移到 after_photo
+                    plugins.setdefault('after_photo', []).append(pinfo)
+                    plugin_list.pop(i)
+                    jm_log('option.migrate',
+                           f'[zip 插件迁移] level 参数已废弃，打包粒度由所在钩子自动推导。'
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)